ts/src/okx.ts

```diff
@@ -2298,33 +2298,73 @@ export default class okx extends Exchange {
          * @param {float} amount how much of the currency you want to trade in units of the base currency
          * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
          * @param {object} params extra parameters specific to the okx api endpoint
+         * @param {bool} params.stop True if is algo order
```

Comment:
We should add here all the other custom parameters such as `takeProfitPrice`, `stopLossPrice`, etc